### PR TITLE
added mit license badge

### PR DESCRIPTION
**Pull request by alexweininger**
Added the MIT license badge to the end of the badge list.

Badge from https://shields.io/

image

### CODE REVIEW

**Comment by EricJizbaMSFT** [comment]
@alexweininger can you link to other repos that have this badge? The License is already linked in other places on both GitHub and the VS Code marketplace so it seems unnecessary to me, but I'm open to adding it if you have other examples of people using it.

# Github:
screen shot 2018-10-19 at 10 23 16 am

# Marketplace in the browser:
screen shot 2018-10-19 at 10 22 05 am

# Marketplace inside VS Code:
screen shot 2018-10-19 at 10 25 56 am

**Comment by alexweininger** [comment]
Here is is on the [Vue repo](https://github.com/vuejs/vue).
image

[React repo](https://github.com/facebook/react)
image

I realize it might seem redundant, but I don't think having the license badge hurts anything.

Thanks!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Azure Functions for Visual Studio Code (Preview)
 
-[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-azurefunctions.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-azurefunctions.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) [![Build Status](https://travis-ci.org/Microsoft/vscode-azurefunctions.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-azurefunctions)
+[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-azurefunctions.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-azurefunctions.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) [![Build Status](https://travis-ci.org/Microsoft/vscode-azurefunctions.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-azurefunctions) ![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)
```
Comment (EricJizbaMSFT) [inline README.md]: Okay I'm fine with the badge. That being said there needs to be a link. For example if you click the travis badge it takes you to Travis CI. If you click the other badges they take you to the VS Code marketplace. Clicking this badge should probably just take you here:
https://github.com/Microsoft/vscode-azurefunctions/blob/master/LICENSE.md

Comment (alexweininger) [reply README.md]: Agreed. Made the badge a link to the license.

**Pull request merged**